facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -137,15 +146,33 @@
 }
 
 @renderContentContainer(collection: EmailContentContainer, collectionIndex: Int) = {
-    @containerTitle(if(collectionIndex > 0) Seq("ct--not-top") else Seq()) {
+    @containerTitle((if(collectionIndex > 0) Seq("ct--not-top") else Nil) ++ (if (collection.branding.isDefined) Seq("ct-branded") else Nil)) {
         @collection.displayName
     }
 
+    @for(containerBranding <- collection.branding) { @containerBranding match {
+        case branding: Branding => {
+            <table class="ct__inner branded-banner">
+                <tr class="bb__inner">
+                    <td class="left-col">
+                    </td>
+                    <td class="bb__label">
+                        Paid for by
```

Comment:
👍 I've switched to using `branding.logo.label` but have moved the branding bar to a separate template to reduce the noise in emailFrontBody